css/templates/e-commerce/page-vars.css

```diff
@@ -15,12 +15,27 @@
  */
 
 :root {
+  --h1: 3.5rem;
```

Comment:
Yeah it's on the high side but we did start with 11! We can review these sizes once all pages have been developed, maybe consolidating some.